### ISSUE TITLE
rake html does not work

### ISSUE THREAD

**Issue by grosser** [runpaint/read-ruby#45]
```
be rake html
/home/micha/.rvm/gems/ruby-1.9.2-preview1/gems/bundler-1.0.0/lib/bundler/shared_helpers.rb:100:in `block in cripple_rubygems': rake is not part of the bundle. Add it to Gemfile. (Gem::LoadError)
    from /home/micha/.rvm/gems/ruby-1.9.2-preview1/bin/rake:18:in `<main>'
[/apps/read-ruby (master)] ➔ rake html
(in /apps/read-ruby)
mkdir -p out
mkdir -p build
rake aborted!
uninitialized constant IO::NULL
/apps/read-ruby/Rakefile:88:in `block (2 levels) in <top (required)>'
(See full trace by running task with --trace)
[/apps/read-ruby (master)] ➔ 
```

**Comment by runpaint** [comment]
`rake` shouldn't be part of the bundle because it's distributed with 1.9.2; installing the gem in addition is inadvisable. `IO::NULL` is a 1.9.3 feature; I'll fix that.

**Comment by runpaint** [comment]
Gemfile: Add 'rake' as dependency. Closed by 23827f6d2e313c083548900cfeb28cb241ce6284

This shouldn't be necessary, as I note in
http://github.com/runpaint/read-ruby/issues#issue/45 , but let's try it anyway
